antora-ui-camel/src/js/vendor/algoliasearch.bundle.js

```diff
@@ -4,8 +4,8 @@
   const algoliasearch = require('algoliasearch/lite')
 
   window.addEventListener('load', () => {
-    const client = algoliasearch('BH4D9OD16A', '16e3a9155a136e4962dc4c206f8278bd')
-    const index = client.initIndex('apache_camel')
+    const client = algoliasearch('XW4Y7DJE2Z', '2423e9780121b010cad442bfb45a4714')
+    const index = client.initIndex('camel_test')
```

Comment:
Do we want to call it "test"?

Reply:
These two lines of changes will be reverted before merging. "camel_test" is the name of my temporary local index which I have linked here so that everyone can see the improved search results.

Reply:
> Do we want to call it "test"?

This is against the test index configuration so we can see the different index configuration in the preview, before we merge we'll revert to the production values. That reminds me we also need to create a PR with the configuration change at Algolia. @aashnajena, @AemieJ do we need to merge the configuration change at Algolia before we merge this change?

Reply:
This PR is so that everyone can search for keywords on the [preview](https://deploy-preview-423--camel.netlify.app/) and comment on whether we are getting better results than before (in terms of grouping results, duplicates, versions, etc).

Reply:
@zregvart Yes, we need to make a PR on Algolia's repository first. If you're satisfied with the current search results, should I go ahead and make that PR?